runtime/kusama/src/lib.rs

```diff
@@ -1044,6 +1045,9 @@ impl InstanceFilter<RuntimeCall> for ProxyType {
 						RuntimeCall::FastUnstake(..)
 				)
 			},
+			ProxyType::NominationPools => {
+				matches!(c, RuntimeCall::NominationPools(..) | RuntimeCall::Utility(..))
```

Comment:
Probably he took it for the same reason that `Utility` is part of `Staking` proxy as well. 

Isn't this an issue that now with Staking proxy you can do `utility::batch[transfer]`?

Reply:
I tried to test this and the batched calls itself are filtered again. So a proxy can only do `utility::batch[some_nomination_call]` if they are only allowed delegation for `NominationPools` .

Perhaps there are some utility calls that are unsafe (.i.e. does not pass along the filters)? It looks to me we are already supporting all utility calls in few other proxies as well.

Reply:
Yes, the batched calls are itself checked again!

Something like aw_derivative is maybe not that good? I would have said that we only allow batching calls.

Reply:
In either case, this is beyond this PR per se, I suggest we move on and revise including `Utility` elsewhere.

Reply:
Not sure how this can be beyond this pr? This is a security issue if not done correctly.

Reply:
Okay, checked the code again `as_derivative` is taking the same filters into account. So, it is safe.

Reply:
we simply always allow Utility because why not? https://github.com/AcalaNetwork/Acala/blob/4844051ecf24c907a21f2dab0145398f50fede4f/runtime/karura/src/lib.rs#L1295